src/sequences/tape.cpp

```diff
@@ -0,0 +1,103 @@
+#include "sequences/tape.h"
+
+#if DEVICE()
+#include <phys253.h>
+#include <LiquidCrystal.h>
+#endif  // marco DEVICE()
+
+namespace sequences
+{
+Tape::Tape()
+{
+  gain_t_ = GAIN_T();
+  gain_p_ = GAIN_P();
+  gain_i_ = GAIN_I();
+  gain_d_ = GAIN_D();
+  velocity_ =  VELOCITY();
+#if USE_UPDATE()
+  state_ = 0;
+#endif  // USE_UPDATE()
+}
+
+void Tape::setup (hardware::Ir ir, hardware::Driver motor)
+{
+  ir_ = ir;
+  motor_ = motor;
+}
+
+bool Tape::loop()
+{
+  error_ = ir_.getTapeError();
+  command_ = computeCommand(error_, 100);
+#if DEBUG() 

```

Comment:
might have to figure out how we want to set the debugs because we have a limited amount of screen space